mavnative/mavnative.c

```diff
@@ -322,7 +322,11 @@ static void set_pyerror(const char *msg) {
 }
 
 // Pass assertion failures back to python (if we can)
+#ifdef __GLIBC__
```

Comment:
Will this cause issues for people building on e.g. MacOSX where they may not be linking against glibc?

Is there a define we can use here explicitly for musl rather than using one for glibc?